### ISSUE TITLE
Training MusicVAE: model not learning

### ISSUE THREAD

**Issue by dubreuia** [tensorflow/magenta#1641]
See https://groups.google.com/a/tensorflow.org/forum/#!topic/magenta-discuss/6r6IFBpP4n8.

The model doesn't train in 1.2.0, I'll try bisecting.

![Screenshot from 2019-12-04 15-24-08](https://user-images.githubusercontent.com/657731/70218965-4caf6700-1744-11ea-83ca-197b1b57196f.png)

**Comment by adarob** [comment]
@dubreuia did you have any luck bisecting?

**Comment by dubreuia** [comment]
I tried in January but didn't go far because my Python environment was getting messed up by the version changes. I'll need to train a MusicVAE model at some point so I'll get back to it, I don't know when.

**Comment by vigliensoni** [comment]
I tried different environments and got the same behavior: the loss decreases on CPU but remains pretty much the same using GPU (both in Colab and a P100). All in all, there is no learning using GPU.

I also followed and posted an issue on the [Tensorflow repo](https://github.com/tensorflow/tensorflow/issues/19200#issuecomment-591697944) but got no useful feedback.

I think I gave up with this, but set up a notebook with a minimal working example so that other    people facing the same issue can easily test other approaches:

https://colab.research.google.com/drive/1ZVMYB-UPJIu7ooNYboEKotC6qatHWgBy#scrollTo=INcHdLXkohOp

**Comment by adarob** [comment]
Thanks Gabriel. Did you try with anything other than the nade?

**Comment by vigliensoni** [comment]
Hi Adam, yes I did. I started creating my own configurations (which work on CPU) and then tried simplifying things by going back to the ones that come bundled with MusicVAE (nade and cat-drums_2bar_small).

Thank you.
